src/Twig/Extension/AceEditorExtension.php

```diff
@@ -33,17 +33,14 @@ public function getName(): string
         return 'ace_editor';
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function getFunctions(): array
     {
         return [
             'include_ace_editor' => new TwigFunction(
                 'include_ace_editor',
                 [$this, 'includeAceEditor'],
                 [
-                    'is_safe' => ['html'],
+                    'is_safe'           => ['html'],
```

Comment:
This is the one rule I don't like (but it's just my personal preference, I dont have a strong opinion about it, up to you, it can stay or you can remmove it and keep 1 space always)

Reply:
I can remove it, I added to keep the style for cases like these: https://github.com/norberttech/aceeditor-bundle/blob/5.x/src/Form/Extension/AceEditor/Type/AceEditorType.php#L67-L108

Reply:
removed :)